src/Files.App/ResourceDictionaries/StatusCenterStyles.xaml

```diff
@@ -0,0 +1,367 @@
+﻿<!--  Copyright (c) 2023 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:converters="using:Files.App.Converters"
+	xmlns:wctconverters="using:CommunityToolkit.WinUI.UI.Converters">
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Dark">
```

Comment:
Can you move this after "Light"? This is the pattern we use in the rest of the codebase.